src/main/java/de/mediathekview/mserver/crawler/orf/tasks/OrfLetterPageTask.java

```diff
@@ -0,0 +1,58 @@
+package de.mediathekview.mserver.crawler.orf.tasks;
+
+import de.mediathekview.mserver.base.Consts;
+import de.mediathekview.mserver.crawler.orf.OrfConstants;
+import de.mediathekview.mserver.crawler.orf.OrfTopicUrlDTO;
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.select.Elements;
+
+public class OrfLetterPageTask implements Callable<ConcurrentLinkedQueue<OrfTopicUrlDTO>> {
+
+  private static final Logger LOG = LogManager.getLogger(OrfLetterPageTask.class);
+  
+  private static final String SHOW_URL_SELECTOR = "ul.latest_episodes > li.latest_episode > a";
+          
+  @Override
+  public ConcurrentLinkedQueue<OrfTopicUrlDTO> call() throws Exception {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "Queue" rather than the implementation "ConcurrentLinkedQueue". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)